addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -3245,6 +3250,12 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    _onTopWindowClick(){
+        if (this.toolbar) {
```

Comment:
I see in 16.0 that `_updateToolbar` starts with `if (!this.toolbar) return;`. I don't know in which version this was introduced but in that version we can remove your `if` (in the forward port).

Reply:
Apparently it's the same thing, it seems that it was just a refactor, currently we have if this.options.toolbar which in fact is equal to this.toolbar

Reply:
Ah yes I see `this.toolbar = this.options.toolbar;` lol!

Yes you can remove the condition then!